### PR TITLE
Pd 892 investigate reporting changes including graphite

### PR DESCRIPTION
**Pull request by DjP-iX**
Creates documentation for the new Reporting Exporters screen/functionality.
Note: I included sectionsfor Graphite as it is the only currently supported export in the TN UI, but Netdata supports a number of other 3rd party exports and the plan is for TN to eventually support multiple Exporter types. So at that point we are probably going to need sections for the different types--this sets up the format to add those when needed. 

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALEUIReference/ReportingScreensSCALE.md [hunk 1]
```diff
@@ -71,31 +79,77 @@ For example, a system with 18,446,744,073,709,551,615 bytes reports the number a
...
+### Add Reporting Exporter
+
+Use the **Add Reporting Exporter** screen to configure third party reporting integrations.
+
+{{< trueimage src="/images/SCALE/23.10/AddReportingExporter.png" alt="Add Reporting Exporter" id="Add Reporting Exporter" >}}
+
+{{< truetable >}}
+| Setting | Description |
+|---------|-------------|
+| **Name** | Enter a unique name for the exporter configuration. If multiple instances are configured, each must have a distinct name. |
```
Comment (micjohnson777) [inline content/SCALEUIReference/ReportingScreensSCALE.md:128]:
```suggestion
| **Name** | Enter a unique name for the exporter configuration. If configuring multiple instances, give each a distinct name. |
```

File: content/SCALEUIReference/ReportingScreensSCALE.md [hunk 2]
```diff
@@ -71,31 +79,77 @@ For example, a system with 18,446,744,073,709,551,615 bytes reports the number a
...
+| **Type** | Select the exporter type to configure. The only current supported option is **GRAPHITE**. |
+| **Enable** | Select to send reporting data to the configured exporter. Deselect to disable the explorer without removing configuration. |
```
Comment (micjohnson777) [inline content/SCALEUIReference/ReportingScreensSCALE.md:130]:
```suggestion
| **Enable** | Select to send reporting data to the configured exporter. Leave checkbox clear to disable the explorer without removing configuration. |
```

File: content/SCALEUIReference/ReportingScreensSCALE.md [hunk 3]
```diff
@@ -71,31 +79,77 @@ For example, a system with 18,446,744,073,709,551,615 bytes reports the number a
...
+
+{{< trueimage src="/images/SCALE/23.10/AddReportingExporterGraphite.png" alt="Graphite Settings" id="Graphite Settings" >}}
+
+{{< truetable >}}
+| Setting | Description |
+|-----------|-------------|
+| **Destination Ip** | Required. Enter the IP address of the Graphite server. |
+| **Destination Port** | Required. Enter the port the Graphite server monitors. |
+| **Prefix** | Enter the prefix to add to all Netdata metrics sent to Graphite. Defaults to *dragonfish*. |
+| **Hostname** | Enter the hostname to add to all metrics, after the prefix, for sending data to the Graphite server. Defaults to *truenas* |
```
Comment (micjohnson777) [inline content/SCALEUIReference/ReportingScreensSCALE.md:145]: This sentence is awkward, like you had a thought, then got distracted. Not sure how to parse it.
```suggestion
| **Hostname** | Enter the host name to add to all metrics, after the prefix, for sending data to the Graphite server. Defaults to *truenas* |
```

File: content/SCALEUIReference/ReportingScreensSCALE.md [hunk 4]
```diff
@@ -71,31 +79,77 @@ For example, a system with 18,446,744,073,709,551,615 bytes reports the number a
...
+| **Update Every** | Enter the interval to send data to the Graphite database, in seconds. Defaults to *1*. |
```
Comment (micjohnson777) [inline content/SCALEUIReference/ReportingScreensSCALE.md:146]:
```suggestion
| **Update Every** | Enter the number of sections for the interval to send data to the Graphite database. Defaults to *1*. |
```

**Review by micjohnson777 — CHANGES_REQUESTED** [review]
Found a few things that needed tweeking. In general, excellent!

File: content/SCALETutorials/ConfigReportsScale.md [hunk 5]
```diff
@@ -36,4 +36,43 @@ Click on the <i class="fa fa-search-minus" aria-hidden="true"></i> icon to zoom
...
+Select the target database from **Type**.
+Graphite is the only current supported option.
+Additional settings fields automatically populate to configure the selected exporter type.
+
+Select **Enable** to send reporting metrics to the configured exporter instance. Deselect to disable the exporter without removing configuration.
+
+### Configuring Graphite Exporting
+
+[Graphite](https://graphiteapp.org/) is a monitoring tool that stores and renders time-series data based on a plaintext database.
+Netdata exports reporting metrics to Graphite in the format `prefix.hostname.chart.dimension`.
```
[the remark this answers is not in the document]
Comment (DjP-iX) [reply content/SCALETutorials/ConfigReportsScale.md:64]:
```suggestion
Netdata exports reporting metrics to Graphite in the format prefix.hostname.chart.dimension.
```

File: content/SCALEUIReference/ReportingScreensSCALE.md [hunk 6]
```diff
@@ -71,31 +79,77 @@ For example, a system with 18,446,744,073,709,551,615 bytes reports the number a
...
+{{< expand "Graphite Settings" "v" >}}
+
+{{< trueimage src="/images/SCALE/23.10/AddReportingExporterGraphite.png" alt="Graphite Settings" id="Graphite Settings" >}}
+
+{{< truetable >}}
+| Setting | Description |
+|-----------|-------------|
+| **Destination Ip** | Required. Enter the IP address of the Graphite server. |
+| **Destination Port** | Required. Enter the port the Graphite server monitors. |
+| **Prefix** | Enter the prefix to add to all Netdata metrics sent to Graphite. Defaults to *dragonfish*. |
```
Comment (DjP-iX) [inline content/SCALEUIReference/ReportingScreensSCALE.md:144]:
```suggestion
| **Prefix** | Enter the prefix to add to all Netdata metrics sent to Graphite. Defaults to *dragonfish*. <br><br>Netdata exports reporting metrics to Graphite in the format *prefix*.*hostname*.*chart*.*dimension*. |
```

**Pull request merged**